### DISCUSSION TITLE
Livechart vs Livechart 2

### DISCUSSION THREAD

**Discussion by PScottPaula** [Live-Charts/LiveCharts2#1226] [General]
Hello All, 
I have a product which is developed in C# WPF. I have requirements to use Charting components. Earlier I used Datavisualization chart which is very old . I need some interactive chart. I saw Livechart and Livechart 2 as well. Kindly suggest me which one I should use and which is more stable with support. Thanks in advance.

**Comment by beto-rodriguez** [comment]
v0 is no longer maintained, I would highly recommend you to use v2, it has a lot of improvements, v2 now supports all the features in v0, and there are huge improvements in performance.

**Comment by upswing1** [comment]
I use Livechart 2 in a .net MAUI app and found out that it is very stable.  The features are periodically updated.  It will not stay in beta for long.  You should use LiveChart 2.
